pkg/mover/chart.go

```diff
@@ -15,10 +15,23 @@ import (
 	"helm.sh/helm/v3/pkg/chartutil"
 )
 
-const DefaultRetries = 3
+const defaultRetries = 3
```

Comment:
Why hide this one?

The publicity serves as documentation on how many retries are the default.

Reply:
Good catch, you are right.